digiroad2-viite/src/main/scala/fi/liikennevirasto/viite/util/AssetDataImporter.scala

```diff
@@ -130,56 +129,124 @@ class AssetDataImporter {
 
   }
 
+  implicit val getRoadAddressHistory = new GetResult[RoadAddressHistory] {
+    def apply(r: PositionedResult) = {
+
+      val roadNumber = r.nextLong()
+      val roadPartNumber = r.nextLong()
+      val trackCode = r.nextLong()
+      val discontinuity = r.nextLong()
+      val startAddrM = r.nextLong()
+      val endAddrM = r.nextLong()
+      val startM = r.nextDouble()
+      val endM = r.nextDouble()
+      val startDate = r.nextTimestampOption().map(timestamp => new DateTime(timestamp))
+      val endDate = r.nextTimestampOption().map(timestamp => new DateTime(timestamp))
+      val validFrom = r.nextTimestampOption().map(timestamp => new DateTime(timestamp))
+      val ely = r.nextLong()
+      val roadType = r.nextLong()
+      val linkId = r.nextLong()
+      val userId =r.nextString
+      val x1 = r.nextDouble()
+      val y1 = r.nextDouble()
+      val x2 = r.nextDouble()
+      val y2 = r.nextDouble()
+      val lrmId = r.nextLong()
+      val ajrId = r.nextLong()
+
+      RoadAddressHistory(roadNumber, roadPartNumber, trackCode, discontinuity, startAddrM, endAddrM, startM, endM, startDate, endDate, validFrom, None, ely, roadType, 0,
+        linkId, userId, Option(x1), Option(y1), Option(x2), Option(y2), lrmId, ajrId)
+
+    }
+  }
+
   private def importRoadAddressData(conversionDatabase: DatabaseDef, vvhClient: VVHClient, ely: Int, importOptions: ImportOptions,
```

Comment:
This method is deprecated or can be removed.

Reply:
I will remove it